dvc/command/version.py

```diff
@@ -139,7 +144,7 @@ def _get_dvc_repo_info(repo):
     if repo.root_dir != repo.scm.root_dir:
         return "dvc (subdir), git"
 
-    return "dvc, git"
+    return "dvc + git"
```

Comment:
We need to make this uniform, either we should make it all commas or all plus-es. Could just use comma for now.

Reply:
Same for cache types which seems to use `/` right now.

Reply:
+1 for commas (since / is a path separator)
+1 to check other occurrences